src/metadata_tool/tools/parsers.clj

```diff
@@ -21,35 +21,42 @@
             [hickory.select                       :as sel]
             [metadata-tool.sources.confluence     :as cfl]
             [metadata-tool.config                 :as cfg]
+            [clojure.pprint                       :as pp]
             [metadata-tool.sources.metadata       :as md]))
 
 (def not-nil? (complement nil?))
 
 (defn parse-string
   [s]
-  (-> s
-      (str/replace "&nbsp;" " ")
-      (str/replace "\u00A0" " ")
-      str/trim))
+  (if (empty? s)
+    ""
+    (-> s
+        (str/replace "&nbsp;" " ")
+        (str/replace "\u00A0" " ")
+        str/trim)))
+
+(defn resolve-acronym
+  [string]
+  (if-let [acronym (get (:acronyms (:meetings cfg/config)) string)]
+    acronym
+    string))
```

Comment:
`get` has a third argument which is the default if not found.  so you can replace this whole function with
`(defn resolve-acronym [s] (get (:acronyms (:meetings cfg/config)) s s))`

Reply:
Well spotted, thanks. I'll have to test it first, but does make sense.